src/NuGet.Indexing/GalleryServiceImpl.cs

```diff
@@ -68,9 +68,15 @@ private static void DocumentCountImpl(JsonWriter jsonWriter, IndexSearcher searc
             ResponseFormatter.WriteV2CountResult(jsonWriter, topDocs.TotalHits);
         }
 
-        private static void ListDocumentsImpl(JsonWriter jsonWriter, NuGetIndexSearcher searcher, Query query, string sortBy, int skip, int take, NuGetSearcherManager manager)
+        private static void ListDocumentsImpl(JsonWriter jsonWriter,
+            NuGetIndexSearcher searcher,
+            Query query,
+            string sortBy,
+            int skip,
+            int take,
+            NuGetSearcherManager manager)
         {
-            Query boostedQuery = new RankingScoreQuery(query, searcher.Rankings);
+            Query boostedQuery = new DownloadsBoostedQuery(query, searcher.DocIdMapping, searcher.Downloads, searcher.Rankings);
```

Comment:
Think we can get rid of the Rankings code path/parameter? Or is it still needed?